pcdsdevices/state.py

```diff
@@ -509,21 +509,10 @@ class TwinCATStateConfigOne(Device):
                      doc='The defined state name.')
     setpoint = Cpt(PytmcSignal, ':SETPOINT', io='io', kind='config',
                    doc='The corresponding motor set position.')
-    delta = Cpt(PytmcSignal, ':DELTA', io='io', kind='config',
-                doc='The deviation from setpoint that still counts '
-                    'as at the position.')
     velo = Cpt(PytmcSignal, ':VELO', io='io', kind='config',
                doc='Velocity to move to the state at.')
-    accl = Cpt(PytmcSignal, ':ACCL', io='io', kind='omitted',
```

Comment:
I'm surprised these weren't deemed important enough to keep, but I'm not the one using them.  

Is there a place where this information will live on, after its removal from here?  Would the PLC be the only place I can find out the `:ACCL` PV exists?

Reply:
I've certainly had second thoughts about the removals- simply removing just the pmps info would be enough to "fix" the immediate problem and the rest might be enthusiastic overoptimization.

The PLC would be the only place that this info would be seen afterwards. So, you'd be unable to inspect or change the motion acceleration profile from EPICS.